featuretools/synthesis/deep_feature_synthesis.py

```diff
@@ -518,11 +515,8 @@ def _build_forward_features(self, all_features, parent_entity,
 
             new_f = DirectFeature(f, child_entity)
 
-            if f.expanding:
```

Comment:
is this removed because we now support direct features with multiple outputs?

Reply:
yes